tests/test_ExperimentDefinition.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from ml_experiment.DefinitionPart import DefinitionPart
+from ml_experiment.ExperimentDefinition import ExperimentDefinition
+
+
+def test_ExperimentDefinition():
+
+    # build dummy experiment
+    exp_name = 'dummy_experiment'
+    part_name = 'qrc'
+
+    part = stubbed_DefinitionPart(exp_name, part_name)
+    part.add_sweepable_property('alpha', (2**-i for i in range(3, 8)))
+    part.add_sweepable_property('beta', [0.5, 1.0, 2.0])
+    part.commit()
+
+
+    # load experiment definition
+    version = 0
+    config_ids = [1, 2, 3]
+    seeds = [1, 2]
+
+    exp = stubbed_ExperimentDefinition(exp_name, part_name, version)
+
+    config = exp.get_config(0)
+    assert config == {'alpha': 0.125, 'beta': 0.5, 'id': 0}
+
+    configs = exp.get_configs(config_ids)
+    assert configs == [
```

Comment:
This does make me wonder if we should consider making some guarantees about ordering. I'm leaning towards "yes", since that's also what PyExpUtils does.

We can make that an issue, then do the guaranteed ordering in another PR.

Reply:
Created #11 for this feature!